### PR TITLE
use pre-kcp branch

### PR DESCRIPTION
**Pull request by MatousJobanek**
use `pre-kcp` branch as the target revision for all infra-deployments-related stuff in `pre-kcp` branch :-)

### CODE REVIEW

**Review by Michkov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Steps:
   d) ArgoCD is set to point to your fork and the preview branch
   e) User is switched back to feature branch to create additional changes
 
-If you want to reset your environment you can run the `hack/util-update-app-of-apps.sh https://github.com/redhat-appstudio/infra-deployments.git staging main` to reset everything including your cluster to `https://github.com/redhat-appstudio/infra-deployments.git` and match the upstream config.
+If you want to reset your environment you can run the `hack/util-update-app-of-apps.sh https://github.com/redhat-appstudio/infra-deployments.git staging pre-kcp` to reset everything including your cluster to `https://github.com/redhat-appstudio/infra-deployments.git` and match the upstream config.
```
Comment (amfred) [inline README.md:173]: Not here exactly, but can you add a paragraph about how to maintain changes to staging using the pre-kcp branch?

- pull the pre-kcp branch locally
- make your changes and commit them
- make a pull request to the pre-kcp branch and get those changes reviewed
- test the changes, either via CI on the new pull request, or by doing a local install and running the E2E tests
- use `kubectl apply -f argo-cd-apps/app-of-apps/all-applications-staging.yaml` to apply the changes to the staging cluster (or should we use the update-app-of-apps.sh)
- merge the changes

File: README.md [hunk 2]
```diff
@@ -332,7 +332,7 @@ Then [save and exit the editor](https://vim.rtorr.com/). The updates will be app
 ## For Members and Maintainers
 
 ### How to add yourself as a reviewer/approver
-There is an OWNERS file present in each component folder [like this](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/spi/OWNERS), people mentioned in the file have the respective access to approve/review PR's.
+There is an OWNERS file present in each component folder [like this](https://github.com/redhat-appstudio/infra-deployments/blob/pre-kcp/components/spi/OWNERS), people mentioned in the file have the respective access to approve/review PR's.
```
Comment (amfred) [inline README.md:335]: I don't think this needs to change, does it?  The OWNERS file in main is the source of truth.

**Review by amfred — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MatousJobanek) [reply README.md:335]: don't forget that the "future" `main` branch (currently `kcp`) won't contain SPI component for now. SPI hasn't been fully onboarded to kcp yet.
In addition, I don't want to make this `pre-kcp` branch dependent on the structure of the `main` one

File: README.md (hunk 1 above)
Comment (MatousJobanek) [reply README.md:173]: I didn't see it as essential to mention it here. The "preview" section partially covers it, and it's a basic "Git & ArgoCD" knowledge after all. 
If you want to use `pre-kcp` AppStudio version, then you should use that branch for everything:
* running scripts
* making PRs
* updating your ArgoCD instance

I've been using the `kcp` branch in that way without any issues. 

But feel free to create a PR if you want to add that section anyway.

Comment (amfred) [reply README.md:173]: Yeah, I'll create a PR.  It'll be against this branch.

**Pull request merged**